Formula/cwb3.rb

```diff
@@ -6,6 +6,11 @@ class Cwb3 < Formula
   license "GPL-2.0-or-later"
   head "svn://svn.code.sf.net/p/cwb/code/cwb/trunk"
 
+  livecheck do
+    url "https://sourceforge.net/projects/cwb/"
+    regex(%r{url=.*?/CWB-CL[._-]v?(\d+(?:\.\d+)+)\.t}i)
```

Comment:
```suggestion
    url "https://sourceforge.net/projects/cwb/rss?path=/cwb"
    regex(%r{url=.*?/cwb[._-]v?(\d+(?:\.\d+)+)-src\.t}i)
```

There are a few different directories in the project but we're only interested in files in the `cwb` directory (and its subdirectories), so we can use `.../rss?path=/cwb` to get the RSS feed for only that directory. This helps to reduce the risk of the files that we're interested in being completely pushed out of the RSS feed over time by other files that we're not interested in.

The regex should also be matching files like `cwb-3.4.33-src.tar.gz`, to align with the `stable` URL. The `CWB-CL` files come from the `CWBperl` directory and don't seem to be what we're using in this formula.